script/mafia

```diff
@@ -87,7 +87,7 @@ exec_mafia () {
 
         bin/bootstrap ) || exit $?
 
-      cp "$MAFIA_TEMP/mafia/.cabal-sandbox/bin/mafia" "$MAFIA_PATH_TEMP"
+      cp "$MAFIA_TEMP/mafia/dist/build/mafia/mafia" "$MAFIA_PATH_TEMP"
```

Comment:
hahaha yeah that's why I was confused - didn't know cabal was racing itself on case-insensitive filesystems